spk/python3/Makefile

```diff
@@ -1,10 +1,11 @@
 SPK_NAME = python3
-SPK_SHORT_VERS = 3.5
-SPK_VERS = $(SPK_SHORT_VERS).6
-SPK_REV = 8
+SPK_SHORT_VERS = 3.6
+SPK_VERS = $(SPK_SHORT_VERS).8
+SPK_REV = 9
 SPK_ICON = src/python3.png
 
-DEPENDS  = cross/busybox cross/$(SPK_NAME)
+# Native needed here to create the host-arch PGEN
+DEPENDS  = cross/busybox native/$(SPK_NAME) cross/$(SPK_NAME)
```

Comment:
I find it strange, cross/python3 already depends on native/python3.

Reply:
I agree, which is why the second and third sentence of this PR mention why this is here, and what I thought a possible root cause was.

Reply:
I really appreciate your contributions - thanks.

Reply:
you're absolutely correct: the redundant dependency wasn't necessary.  I added it because I got around a build error by rebuilding `native/python3` but removing this dependency didn't stop a clean build from completing.

I'll update the PR to remove that line.

I've also completed testing and will update the top comment.

Reply:
(updating the PR in a day; rolling in `PLIST_TRANSFORM` to reduce churn on update)

Reply:
(these edits are done as discussed)